### PR TITLE
Convert to c

### LINKED ISSUE #4: Developing convertToC()
TO DO:
- [x] Develop function ```convertToC()```
- [x] Test its behaviour
- [x] Add to the interface

### PR DESCRIPTION
**Pull request by gelilaa**
Closes # 4

### CODE REVIEW

File: scripts/convert-to-c.js [hunk 1]
```diff
@@ -3,7 +3,10 @@
 console.log('-- loading: convertToC');
 
 
-function convertToC() { }
+function convertToC(fahrenheit) {
+  const celsius = (fahrenheit - 32) * 5/9;
```
Comment (pavelbidenko2018) [inline scripts/convert-to-c.js]: Maybe Math. round() could be used to return a rounded result? Cause now -14 returns  6.800000000000001

**Review by pavelbidenko2018 — CHANGES_REQUESTED** [review]
Seems that both buttons 'Convert to C' and 'Convert to F' returns the same value..

File: scripts/convert-to-c.js (hunk 1 above)
Comment (gelilaa) [reply scripts/convert-to-c.js]: Yes I  sow it. the problem was in the function for the convertToCHandler had some spelling mistake.  Fixing it now.

Comment (gelilaa) [reply scripts/convert-to-c.js]: It is done.

**Review by pavelbidenko2018 — APPROVED** [review]

**Review by pavelbidenko2018 — APPROVED** [review]

**Review by pavelbidenko2018 — APPROVED** [review]

**Review by fmkarakus — APPROVED** [review]
Everything works. Good work!

**Review by pavelbidenko2018 — APPROVED** [review]
OK )

**Pull request merged**